docs/using-kontena/nodes.md

```diff
@@ -42,6 +42,36 @@ The re-scheduling of grid services will happen after the node offline grace peri
 - `ha`: 2 minutes
 - `random`: 30 seconds
 
+### Node availability
+
+Node availablity for scheduling services can be controlled with `kontena node availablity` command. The current supported states are: `drain`and `active`.
```

Comment:
oh, I forgot to update the docs in https://github.com/kontena/kontena/pull/2306/commits/a48f420b56d3d66bf77aeb95986f280201af09a1, that changed the availability to use `kontena node update --availability`